src/data/finset/basic.lean

```diff
@@ -1505,6 +1505,9 @@ theorem filter_insert (a : α) (s : finset α) :
   filter p (insert a s) = if p a then insert a (filter p s) else filter p s :=
 by { ext x, simp, split_ifs with h; by_cases h' : x = a; simp [h, h'] }
 
+theorem filter_erase (a : α) (s : finset α) : filter p (erase s a) = erase (filter p s) a :=
+by { ext x, simp [and_assoc]}
```

Comment:
Could you squeeze this simp to match some of the lemmas above?

Reply:
Done!